oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreServiceTest.java

```diff
@@ -155,18 +153,6 @@ public void setUpdateLimit() throws Exception {
         assertEquals(17, store.getUpdateLimit());
     }
 
-    @Test
-    public void keepAlive() throws Exception {
```

Comment:
`keepAlive` config is only marked as deprecated not removed. I would suggest to keep these test till we remove that.

Reply:
This test checks for `getMongoClientOptions().isSocketKeepAlive()`, which doesn't exist at all in modern driver, so this test would have no sense at all.

I marked `keepAlive` and all related properties as deprecated just to help to detect any uses in any application using Oak as dependency, to avoid breaking changes, but all those options should be completely removed.